packages/angular/common/src/directives/navigation/tabs.ts

```diff
@@ -142,7 +178,45 @@ export abstract class IonTabs implements AfterContentInit, AfterContentChecked {
     }
   }
 
+  private setActiveTab(tab: string): void {
+    const tabs = this.tabs;
+    const selectedTab = tabs.find((t: any) => t.tab === tab);
+
+    if (!selectedTab) {
+      console.error(`tab with id: "${tab}" does not exist`);
+      return;
+    }
+
+    this.leavingTab = this.selectedTab;
+    this.selectedTab = selectedTab;
+
+    this.ionTabsWillChange.emit({ tab });
+
+    selectedTab.el.active = true;
+  }
+
+  private tabSwitch(): void {
+    const selectedTab = this.selectedTab;
+    const leavingTab = this.leavingTab;
+
+    if (this.tabBar) {
+      this.tabBar.selectedTab = selectedTab.tab;
+    }
+
+    if (leavingTab?.tab !== selectedTab.tab) {
+      if (leavingTab) {
```

Comment:
I believe this should be checking that the `el` is defined, not the `leavingTab`, since it will based on the check on L206.